Source/WebInspectorUI/UserInterface/Views/TreeOutline.css

```diff
@@ -47,7 +47,7 @@
 .tree-outline .item {
     height: 20px;
 
-    padding: 0 var(--tree-outline-item-padding);
+    padding: 0 0 0 var(--tree-outline-item-padding);
 
```

Comment:
this seems like a **_very_** scary change since it could affect _all_ trees everywhere in Web Inspector

are we sure this is OK for all the various uses?  could we maybe just override this for object trees instead?

Reply:
Whoops, I did not notice that I'm changing `.tree-outline` instead of `.object-tree` at this point. Very scary indeed 😨